README.md

```diff
@@ -112,6 +112,24 @@ GET    /workflow_queue/all_queued
 GET    /workflow_queue
 ```
 
+### Workflow Variables
+
+If a workflow or workflows for a particular object require data to be persisted and available between steps, workflow variables can be set.
+These are per object/version pair and thus available to any step in any workflow for a given version of an object once set.
+
+These data are not persisted in Cocina, and are not preserved or available outside of the workflow-service, so they should only be used to persist information used during workflow processing.
+
+To use, pass in a "metadata" parameter as a hash when creating a workflow. The hash can contain any number of key/value pairs of metadata:
```

Comment:
I'd suggest "context" as a name rather than "metadata".  We already have "metadata" and it means other things. We're overloading that term too much.